modules/mono/editor/GodotTools/GodotTools.ProjectEditor/GodotTools.ProjectEditor.csproj

```diff
@@ -31,14 +31,7 @@
     <Reference Include="System" />
     <Reference Include="Microsoft.Build" />
     <Reference Include="DotNet.Glob, Version=2.1.1.0, Culture=neutral, PublicKeyToken=b68cc888b4f632d1, processorArchitecture=MSIL">
-      <!--
-      When building Godot with 'mono_glue=no' SCons will build this project alone instead of the
-      entire solution. $(SolutionDir) is not defined in that case, so we need to workaround that.
-      We make SCons restore the NuGet packages in the project directory instead in this case.
-      -->
-      <HintPath Condition=" '$(SolutionDir)' != '' ">$(SolutionDir)\packages\DotNet.Glob.2.1.1\lib\net45\DotNet.Glob.dll</HintPath>
       <HintPath>$(ProjectDir)\packages\DotNet.Glob.2.1.1\lib\net45\DotNet.Glob.dll</HintPath>
-      <HintPath>packages\DotNet.Glob.2.1.1\lib\net45\DotNet.Glob.dll</HintPath> <!-- Are you happy CI? -->
```

Comment:
I'm pretty sure without this the CI build kept failing. I have no idea why it's building fine now after removing it...

Reply:
Both give the same result because one is the relative path and the other are absolute. You can test cleaning and rebuilding the entire solution. It's working fine on Windows and the CI. It was previous using two different path: one inside the solution directory and other inside the project directory and it was causing issues depending on the build target.